src/api/v1beta1/properties.go

```diff
@@ -225,6 +234,11 @@ func (dk *DynaKube) ImmutableOneAgentImage() string {
 		return oneAgentImage // TODO: What to do with the Version field in this case ?
 	}
 
+	codeModulesImage := dk.CodeModulesImage()
```

Comment:
do we need this ? @chrismuellner

Reply:
I dont think we need that here, CodeModuleImage should be an own function and has nothing in common with the 'image' field of the osAgent

Reply:
okay, yes that makes sense. what i do not understand currently is, where is CodeModulesImage() then called? Or is it nowhere called for the moment?

Reply:
that's fine, it will be used by the csi driver in the future